### ISSUE TITLE
`configcat flag-v2 value show --json` returns an empty object

### ISSUE THREAD

**Issue by oshaughnessy** [configcat/cli#31]
### Describe the bug

Running `configcat flag-v2 value show --json`, whether or not a `flag-id` is given, produces an empty result.

### To reproduce

```
$ configcat flag-v2 value show --flag-id _some_id_ --json
{}
```

### Expected behavior

A JSON object with data about the state of the flag in each environment, similar to the report-style output that is provided without the `--json` flag.

**Comment by z4kn4fein** [comment]
Hi @oshaughnessy, thank you for reporting this issue! I'm going to investigate what's happening and I'll let you know when the fix is released.

**Comment by oshaughnessy** [comment]
I'm sure you'll have a fix out quick, but if anybody else needs a workaround, this does the trick in bash:

```
function cc_get_env_flag
{
    local _env_name=$1
    local _flag_id=$2
    export _env_name
    configcat flag-v2 value show --flag-id $_flag_id --non-interactive |awk '/^----/ { MATCH=0 } $2==ENVIRON["_env_name"] { MATCH=1 } $2=="Default:" && MATCH==1 { RESULT=$3 } END { print RESULT }'
    export -n _env_name
}
```

Then pass it your environment name and flag ID, e.g.
```
flag_val=$(cc_get_env_flag dev 123456)
echo $flag_val
```

**Comment by z4kn4fein** [comment]
Hey @oshaughnessy, I've released `v2.3.3` with the fix, could you please verify that it works as you expected? Thanks!

**Comment by oshaughnessy** [comment]
That looks great. Thank you!
